README.md

```diff
@@ -1,77 +1,76 @@
-# scikit-learn-knn-regression
+# sknnr
 
-This package is in active development.
+> ⚠️ **WARNING: sknnr is in active development!** ⚠️
 
-## Developer Guide
+## What is sknnr?
 
-### Setup
+`sknnr` is a package for running k-nearest neighbor (kNN) imputation methods using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/). Notably, common methods such as most similar neighbor (MSN, Moeur & Stage 1995), gradient nearest neighbor (GNN, Ohmann & Gregory, 2002), and random forest nearest neighbors (RFNN, Crookston & Finley, 2008) are included in this package.
```

Comment:
Should we use a shortened version of this for the repository "Description"? Maybe something like: 

> kNN imputation methods (MSN, GNN, RFNN, etc.) with scikit-learn compatible estimators

or...

> scikit-learn compatible estimators for kNN imputation

or some variation on that. Seems like a good time to flesh out the repo while we're tackling the readme.

Reply:
Sounds great!  I think I prefer the second variation, but perhaps expanded to be:

> scikit-learn compatible estimators for various kNN imputation methods

That may not be quite right, but some users might think that we've just replicated `KNeighborsRegressor` and `KNeighborsClassifer` if we don't somehow specify that there are different "flavors" to how you set up the distance space.    (I was going to use the word "flavors" but that seemed a bit too casual!)

Reply:
Yeah, good addition! I added that to the repository info as well, as the eventual docs link and some tags (add/edit as you see fit!)

Reply:
I don't feel like I'm seeing the docs link or any tags in either of your latest two commits.  Am I missing them?  I assume the changes would have been to `pyproject.toml`.  I think I've pulled to the latest version.

Reply:
I was just talking about the repository "About" section on the Github home page, but I think you're right that we can/should add those same details to the `pyproject.toml`. I'll go ahead and do that once I figure out where they're supposed to go!

Reply:
Ah, now I see them!  Awesome work!  Tags look good to me as well.